src/coreclr/vm/excep.cpp

```diff
@@ -7360,6 +7360,12 @@ VOID DECLSPEC_NORETURN UnwindAndContinueRethrowHelperAfterCatch(Frame* pEntryFra
         }
         else
         {
+#ifdef FEATURE_INTERPRETER
+            if ((pEntryFrame != NULL) && (pEntryFrame->GetFrameIdentifier() == FrameIdentifier::InterpreterFrame))
```

Comment:
Does both of us overlooking 'pEntryFrame can be NULL' in this case indicate that we need a comment in a key spot to make it clear in the future that it's potentially null?

Reply:
I've actually just pushed another fix, the NULL was not a correct fix. The correct check is FRAME_TOP. The Thread::GetFrame() that is a source of this pEntryFrame returns that when there are no frames on the thread.